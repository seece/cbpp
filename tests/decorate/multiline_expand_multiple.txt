#macro TESTMACRO(a, b)
Log(s:"a is ", d:a);
Log(s:"b is ", d:b);
#endmacro

#macro SPAM(a, b)
Log(s:"a is ", d:a);
Log(s:"a is ", d:a, s:"!");
Log(s:"a is ", d:a, s:"!", d:a, d:a);
#endmacro

TESTMACRO(1,2)
TESTMACRO(3,4)

SPAM(5,6)