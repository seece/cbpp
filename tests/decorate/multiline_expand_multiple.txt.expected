// #macro TESTMACRO(a, b)
// Log(s:"a is ", d:a);
// Log(s:"b is ", d:b);
// #endmacro

// #macro SPAM(a, b)
// Log(s:"a is ", d:a);
// Log(s:"a is ", d:a, s:"!");
// Log(s:"a is ", d:a, s:"!", d:a, d:a);
// #endmacro

Log(s:"a is ", d:1);
Log(s:"b is ", d:2);
Log(s:"a is ", d:3);
Log(s:"b is ", d:4);

Log(s:"a is ", d:5);
Log(s:"a is ", d:5, s:"!");
Log(s:"a is ", d:5, s:"!", d:5, d:5);